config/locales/de.yml

```diff
@@ -823,7 +825,7 @@ de:
       failure: Fehlerhafte E-Mail oder Passwort.
       success: Sie haben sich erfolgreich angemeldet.
     create_through_lti:
-      session_with_outcome: 'Bitte beachten Sie, dass zur Gutschrift der Punkte Ihr Code nach der Bearbeitung durch Klicken auf den Button "Code zur Bewertung abgeben" eingetragen werden muss.'
+      session_with_outcome: 'Ihre Punkte werden durch Klicken auf den Knopf "Bewerten & Abgeben" übermittelt.'
```

Comment:
"Knopf"? Interesting wording ;)

Reply:
Which German word are you using for the English word "button"? "Taste"? Or should I just write: "Ihre Punkte werden durch Klicken auf 'Bewerten & Abgeben' übermittelt."?